### PR TITLE
Virtual piano

### PR DESCRIPTION
**Pull request by VaishnaviMankala19**
Closes: #271

- **Title** : VIRTUAL PIANO
- **Name:** : MANKALA VAISHNAVI
- **Idenitfy yourself:** : gssoc24

### Describe the add-ons or changes you've made 📃
I have added Virtual-Piano in Vanilla.js>basic , updated readme and database

### Screenshots 📷

Screenshot (24)

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: Vanilla-JS-Projects/Basic/Virtual-piano/output.png
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Virtual-piano/output.png:1]: ss name is to be `screenshot.webp` i.e. not in png format it is required in webp format and change the name as it is.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@VaishnaviMankala19 LGTM. But change the directory named to `Virtual-Piano`

**Comment by VaishnaviMankala19** [comment]
Can i get level2 for this @Avdhesh-Varshney

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@VaishnaviMankala19 change `Virtual-piano` folder name to `Virtual-Piano` and resolve the merge conflicts.

**Comment by VaishnaviMankala19** [comment]
@Avdhesh-Varshney please review my pr

**Comment by Avdhesh-Varshney** [comment]
@VaishnaviMankala19 again raise your pr with proper guidelines. It can't be approved as there are 34 commits for a single project addition.

**Pull request closed without merging**